kernel/build-kernel.sh

```diff
@@ -391,31 +418,35 @@ install_kata() {
 
 	install --mode 0644 -D ./.config "${install_path}/config-${kernel_version}"
 
-	if [[ ${experimental_kernel} == "true" ]]; then
-		sufix="-virtiofs.container"
-	else
-		sufix=".container"
-	fi
-
-	ln -sf "${vmlinuz}" "${install_path}/vmlinuz${sufix}"
-	ln -sf "${vmlinux}" "${install_path}/vmlinux${sufix}"
-	ls -la "${install_path}/vmlinux${sufix}"
-	ls -la "${install_path}/vmlinuz${sufix}"
+	ln -sf "${vmlinuz}" "${install_path}/vmlinuz${sufix}.container"
+	ln -sf "${vmlinux}" "${install_path}/vmlinux${sufix}.container"
+	ls -la "${install_path}/vmlinux${sufix}.container"
+	ls -la "${install_path}/vmlinuz${sufix}.container"
 	popd >>/dev/null
 }
 
 main() {
-	while getopts "a:c:hk:p:t:v:e" opt; do
+	while getopts "a:c:dfghk:p:t:v:e" opt; do
 		case "$opt" in
 			a)
 				arch_target="${OPTARG}"
 				;;
 			c)
 				kernel_config_path="${OPTARG}"
 				;;
+			d)
+				PS4=' Line ${LINENO}: '
```

Comment:
sweet ;-)

Reply:
PS5 is coming :wink:

Reply:
The PS4 defined the following two codes: $0 and $LINENO.